### PR TITLE
wordy: update tests to v1.2.0

### LINKED ISSUE #1530: wordy: update tests to v1.2.0
The [test suite](https://github.com/exercism/python/blob/master/exercises/wordy/wordy_test.py) for wordy is out of date and needs updated to conform to the [latest canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/wordy/canonical-data.json).

### PR DESCRIPTION
**Pull request by fortrieb**
Update wordy test to 1.2.0. Fix #1530

### CODE REVIEW

File: exercises/wordy/wordy_test.py [hunk 1]
```diff
@@ -51,12 +51,14 @@ def test_multiple_division(self):
             calculate("What is -12 divided by 2 divided by -3?"), 2)
 
     def test_unknown_operation(self):
-        with self.assertRaisesWithMessage(ValueError):
+        with self.assertRaisesWithMessage(ValueError) as error:
```
Comment (cmccandless) [inline exercises/wordy/wordy_test.py]: In this track, we don't test for actual exception message content, only that one is present. The reasoning for this is summarized well here: https://github.com/exercism/python/issues/1080#issuecomment-342013223

Comment (fortrieb) [reply exercises/wordy/wordy_test.py]: Ok. Well should i check that the error message is not empty? Or is raising an error enough?

Comment (cmccandless) [reply exercises/wordy/wordy_test.py]: `assertRaisesWithMessage` checks for a non-null, non-empty error message.

**Comment by cmccandless** [comment]
Merged; thanks for working on this!

**Pull request merged**